### PR TITLE
Completed html-syntax excercise.

### PR DESCRIPTION
**Pull request by phong4**
Screen Shot 2021-11-29 at 1 06 28 PM

### CODE REVIEW

File: html-syntax/index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+  <img src = "https://static.wikia.nocookie.net/spellcasters/images/e/e3/145Zapdos.png/revision/latest?cb=20210101111145">
+  <h1>
+    Zapdos
+  </h1>
```
Comment (codingcodymiller) [inline html-syntax/index.html]: These should be in the `body` element, they are visual content. Additionally, you are missing your `head` element, please include that as well, even if it is empty

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: html-syntax/index.html (hunk 1 above)
Comment (phong4) [reply html-syntax/index.html]: Thanks @codingcodymiller I have made the changes. Please let me know if theres anything else I need to modify.

File: html-syntax/index.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Syntax</title>
+  </head>
+  <body>
+  <img alt = "Zapdos" src = "https://static.wikia.nocookie.net/spellcasters/images/e/e3/145Zapdos.png/revision/latest?cb=20210101111145">
```
Comment (codingcodymiller) [inline html-syntax/index.html]: No spaces around equal signs for attributes, please

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by phong4** [comment]
I removed the spaces around equal signs for attributes. Please let me know if I'm missing anything else! @codingcodymiller

**Review by codingcodymiller — APPROVED** [review]
Great work!

**Pull request merged**